cmake/cmake_common_scripts-config.cmake.in

```diff
@@ -4,3 +4,23 @@ set(@PROJECT_NAME@_FOUND ON)
 
 include("${CMAKE_CURRENT_LIST_DIR}/cmake_tools.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/code_coverage.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/gtest.cmake")
```

Comment:
In this case, this does not need to be here since it includes the building of GTest which you do not want to happen during a find_package call. Only the information below should be need by the user since GTest should already be built.